app/components/home/Challenge.tsx

```diff
@@ -46,6 +47,12 @@ export function Challenge({
     const fetchUser = async () => {
       try {
         const userData = await firestoreCtrl.getUser(challengeDB.uid);
+        if (
+          userData.image_id?.startsWith("http") ||
```

Comment:
Why use an insecure protocol?

Reply:
The creation of the URL doesn't depend entirely on us but more on the firebase methods so it seems it should be there still.